src/main/kotlin/no/nav/syfo/dialogmote/database/ReferatQuery.kt

```diff
@@ -182,19 +261,14 @@ fun Connection.createNewReferat(
             it.executeQuery()
         }
     }
-
-    if (commit) {
-        this.commit()
-    }
-    return Pair(referatId, referatUuid)
 }
 
 const val queryGetReferatWithoutJournalpostArbeidstaker =
     """
         SELECT MOTEDELTAKER_ARBEIDSTAKER.PERSONIDENT, MOTE_REFERAT.*
         FROM MOTE INNER JOIN MOTE_REFERAT ON (MOTE.ID = MOTE_REFERAT.MOTE_ID)
                   INNER JOIN MOTEDELTAKER_ARBEIDSTAKER ON (MOTE.ID = MOTEDELTAKER_ARBEIDSTAKER.MOTE_ID) 
-        WHERE MOTE_REFERAT.journalpost_id IS NULL
+        WHERE MOTE_REFERAT.journalpost_id IS NULL AND MOTE_REFERAT.ferdigstilt = true
```

Comment:
Bare ferdigstilte referater som skal journalføres

Reply:
Kan også rename query-variabelen (og evt metoden som bruker den) for å tydeliggjøre at bare ferdigstilte referater hentes

Reply:
Fikset